config.json

```diff
@@ -134,10 +134,10 @@
         "storagePrefix": "session/"
     },
     "server": {
-        "baseUrl": "https://api.opentoken.io",
+        "baseUrl": "http://localhost",
```

Comment:
I upload a different config.  Specifically, I use the `override.json.gpg` when uploading.